docs/bucket/notifications/README.md

```diff
@@ -175,10 +176,13 @@ An example configuration for MQTT is shown below:
         "qos": 1,
         "clientId": "minio",
         "username": "",
-        "password": ""
+        "password": "",
+        "queueDir": ""
     }
 }
 ```
+MQTT supports persistent event store to back up events incase if the mqtt broker goes down. The event store can be configured by setting the `queueDir` field in the mqtt config.
```

Comment:
Suggestion: Could we elaborate this by saying that MQTT supports persistent event store in the client-side or publiser-side to be more precise?

Reply:
yes makes sense. Will do